virttest/guest_agent.py

```diff
@@ -897,7 +897,7 @@ def ssh_add_authorized_keys(self, username, *keys, **kwargs):
         cmd = "guest-ssh-add-authorized-keys"
         keys = list(keys)
         reset = kwargs.get('reset')
-        return self.cmd_args_update(cmd, username=username,
+        return self._cmd_args_update(cmd, username=username,
                                     keys=keys, reset=reset)
```

Comment:
This line should be updated too, to make new alignment.

Reply:
oh,yes.